src/pystatis/cache.py

```diff
@@ -132,7 +132,7 @@ def normalize_name(name: str) -> str:
     Returns:
         str: The unique identifier without the optional job id.
     """
-    if len(re.findall(JOB_ID_PATTERN, name)) == 3:
+    if re.findall(JOB_ID_PATTERN, name):
```

Comment:
The original idea was to parse a number like `51000-0017_988038177` and omit the last part so that the job id becomes the original evas number. Your code is no longer checking for valid job ids but only for any numbers. I think it still works because `split` just returns the input if it cannot find a "_", but the logic is now different and the pattern is no longer a JOB_ID_PATTERN so we should rename this constant if you did this deliberately

Reply:
The previous approach only worked for GENESIS table codes because it merely checked if there were three groups of numbers (Regionalstatistik and Zensus tables have letters and varying numbers of number groups). The underscore, however, is specific to the Job ID generated when a table is requested as a background job, so it is safer to check it this way. I am not attached to the name JOB_ID_PATTERN but in my interpretation, the job id pattern is characterized by the underscore and following numbers, so it makes sense to me to keep the name. I am happy to change it though if you disagree.